README.MD

```diff
@@ -1,5 +1,7 @@
 # Kafka Proto Parquet Writer
-Kafka proto parquet writer reads records from a Kafka topic and writes them as parquet files. Parquet files can be created on local file system or HDFS. It can read and write records in multiple threads. As writing to single parquet file can not be done concurrently, each threads writes to a separate file. 
+Kafka proto parquet writer reads records from a Kafka topic and writes them as parquet files. Parquet files can be created on local file system or HDFS. It can write records in multiple threads. As writing to single parquet file can not be done concurrently, each threads writes to a separate file. 
+
+Kafka proto parquet writer uses [Smart Commit Kafka Consumer](https://github.com/sahabpardaz/smart-commit-kafka-consumer) module for reading records from kafka.
 
```

Comment:
At least once delivery is guaranteed because the consumer will be notified of a record's ack just if it is written in a parquet file and successfully flushed to the disk.